exec/java-exec/src/test/java/org/apache/drill/exec/fn/impl/TestCastFunctions.java

```diff
@@ -30,6 +31,11 @@
 
 public class TestCastFunctions extends BaseTestQuery {
 
+  @BeforeClass
+  public static void setupLocal() {
+    System.setProperty("user.timezone","Etc/GMT");
```

Comment:
We should change the timezone in the same way as I proposed for the locale.
Also since we fix test failures connected with the timezone, please update Jira and pull request titles.